File: php/src/Test/Classic/MyClassTest.php
<?php

namespace Foo\Test\Classic;

use Foo\Classic\MyClass;

class MyClassTest extends \PHPUnit_Framework_TestCase
{

    /**
     * @test
     */
    public function shouldReturnTrueMyMethod()
    {
        $myClass = new MyClass();

        $result = $myClass->myMethod();

        $this->assertTrue($result);
    }
}

File: java/test/Classic/MyClassTest.java
package Classic;

import static org.junit.Assert.assertTrue;

import org.junit.Before;
import org.junit.Test;

public class MyClassTest {

	@Before
	public void setUp() throws Exception {
	}

	@Test
	public void testClassWorking() {
		MyClass myClass = new MyClass();
		assertTrue(myClass.myFirstTestMethod());
	}

}
